### DISCUSSION TITLE
What's the point of VERDACCIO_USER_NAME and VERDACCIO_USER_UID?

### DISCUSSION THREAD

**Discussion by OnkelTem** [verdaccio/verdaccio#2341] [Q&A] [answered]
Setting these variables in `docker-compose.yml`:

```yaml
    environment:
      - VERDACCIO_USER_NAME=node
      - VERDACCIO_USER_UID=1000
```

doesn't seem to affect anything. The main process is still running under `verdaccio` user with id of `10001`.

I wonder what's the point of using them then? Which use case do they cover?

It's a follow-up of: https://github.com/verdaccio/verdaccio/issues/2268

**Comment by juanpicado** [comment, marked as answer]
I think is only useful on extend images, examples, install your own plugins. For runtime only the protocol one is dynamic.
